packages/manager/cypress/e2e/core/stackscripts/delete-stackscripts.spec.ts

```diff
@@ -0,0 +1,104 @@
+import { authenticate } from 'support/api/authentication';
+import { stackScriptFactory } from 'src/factories';
+import {
+  mockDeleteStackScript,
+  interceptGetStackScripts,
+  mockGetStackScripts,
+} from 'support/intercepts/stackscripts';
+import { ui } from 'support/ui';
+
+authenticate();
+describe('Delete stackscripts', () => {
+  /*
+   * - Deletes the stackscripts.
+   * - Confirms that the stackscript item still exist when cancelling the delete operation.
+   * - Confirms that the stackscript item can be deleted successfully.
+   * - Confirms that "Automate Deployment with StackScripts!" welcome page appears when user has no StackScript.
+   */
+  it('deletes the stackscripts', () => {
+    const stackScripts = stackScriptFactory.buildList(2, {
+      is_public: false,
+    });
+    interceptGetStackScripts(stackScripts).as('getStackScripts');
```

Comment:
```suggestion
    mockGetStackScripts(stackScripts).as('getStackScripts');
```
I think this may be the cause of the test failure!